monitor/config_test.go

```diff
@@ -0,0 +1,162 @@
+package monitor
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gofiber/fiber/v3"
+)
+
+func Test_Config_Default(t *testing.T) {
```

Comment:
Replace all the `AssertEqual()` calls with `require.Equal()` from https://github.com/stretchr/testify

Reply:
[b5c4678b7d76ec59aae75fd0e492f0388e6ab5d8](https://github.com/Redish101/contrib/commit/b5c4678b7d76ec59aae75fd0e492f0388e6ab5d8)